weblab/templates/home.html

```diff
@@ -1,5 +1,72 @@
 {% extends "base.html" %}
+{% load entities %}
+{% load staticfiles %}
 
 {% block content %}
   <h1>Cardiac Electrophysiology Web Lab</h1>
+
+  <p>
+    This is the beta version of 'Web Lab 2', a second generation Web Lab incorporating experimental data and parameter fitting.
+    You can read more about our plans in a <a href="https://www.biorxiv.org/content/early/2018/01/31/257683">preprint of our next Web Lab paper</a>.
+  </p>
+
+  <h2>Quick start links</h2>
+  <ul>
+    <li><a href="{% url 'experiments:list' %}">View results of experiments</a> stored on this site.</li>
+    <li>Compare the results of different experiments, e.g. ...; or <a href="{% url 'experiments:list' %}">set up your own comparisons</a>.</li>
+    {% if user.is_authenticated %}
+      {% url 'entities:models' as analyse_link %}
+    {% else %}
+      {% url 'accounts:register' as analyse_link %}
+    {% endif %}
+    <li><a href="{{ analyse_link }}">Analyse your own models/protocols</a>.</li>
+    <li>Find out more about this site - read on!</li>
+  </ul>
+
+  <h2>What is the Web Lab?</h2>
+  <p>
+    Modellers have adopted XML-based markup languages to describe mathematical models over the past decade.
```

Comment:
languages (e.g. CellML)

The only SBML electrophysiology model I've seen is a Hodgkin-Huxley implementation

Reply:
We talk specifically about CellML further down. I'd rather keep this general, as the Web Lab can be used for non-cardiac domains too.